sopel/test_tools.py

```diff
@@ -259,3 +272,257 @@ def run_example_tests(filename, tb='native', multithread=False, verbose=False):
         args.extend(["-n", str(cpu_count())])
 
     pytest.main(args)
+
+
+def botfactory(settings, autoloads=None):
+    """Build a test instance of Sopel, with a :class:`MockIRCBackend`.
+
+    :param settings: Sopel's configuration for testing purposes
+    :type settings: :class:`sopel.config.Config`
+    :param list autoloads: list of plugins to autoload after creating the bot
+    :return: a test instance of the bot
+    :rtype: :class:`sopel.bot.Sopel`
+
+    This will instanciate a :class:`~sopel.bot.Sopel` object, replace its
```

Comment:
```suggestion
    This will instantiate a :class:`~sopel.bot.Sopel` object, replace its
```

With a 'c' is an alternative form that isn't used much. Chrome's spell-checker doesn't even recognize it (but I know it's considered correct in some regions, technically).